tests/example.py

```diff
@@ -73,9 +73,9 @@ def testCore():
 
     print('\nCreating Queries\n')
     # 4 - Create queries
-    clsRadioFluxF15.operate('time<0')
-    sorceMGIndex.operate('time<2452705')
-    clsRadioFluxF107.operate('time,absolute_f107').operate('time<1952').operate('formatTime(yyyy.MM.dd)')
+    clsRadioFluxF15.addOperation('time<0')
+    sorceMGIndex.addOperation('time<2452705')
+    clsRadioFluxF107.project(['time,absolute_f107']).select(rangeStart='1953', rangeEnd='1954').select(target='absolute_f107', rangeEnd='70').addOperation('formatTime(yyyy.MM.dd)')
```

Comment:
Related to my comment about refactoring `project()`, notice how you're not giving it a list of two strings here. `'time,absolute_f107'` is one string. This example would break if the list actually had multiple strings.